README.md

```diff
@@ -120,6 +120,11 @@ jlpm run build
 jupyter lab build
 ```
 
+Or start in watch mode. Then after ever change, refresh the page and click "build".
+```bash
+jlpm run watch
+```
```

Comment:
How about also documenting the steps to launch jupyterlab in watch mode?

```
jupyter-lab --watch
```

Reply:
Done. Watch is better but somewhat frustrating due to the time it takes so I don't always know if my change is really there.